meadow-client/src/MainFrame.purs

```diff
@@ -287,12 +287,11 @@ applyTransactionM oldState =
     Nothing -> oldState
     Just c -> case applyTransaction inps sigs bn st c mic of
                 MSuccessfullyApplied {funds, state, contract} _ ->
-                   set (_transaction <<< _inputs) []
-                   (set (_transaction <<< _signatures) Map.empty
-                   (set (_state) state
-                   (set (_moneyInContract) funds
-                   (set (_contract) (Just contract)
-                    oldState))))
+                   set (_transaction <<< _inputs) [] oldState
```

Comment:
Minor thing, so feel free to dismiss this as me being fussy, but I like to format these sorts of blocks as:

```
oldState
# set (_transaction <<< _inputs) []
# set (_transaction <<< _signatures) Map.empty
...
```

It's an Elm-ish thing that I think reads neatly.

Reply:
and a Clojure thing, I actually thought of this later but was too lazy to change it, I'll do it now, I'm motivated!

Reply:
also, I wish `#` was `|>`